SourceKitStressTester/Sources/StressTester/ActionGenerators.swift

```diff
@@ -439,6 +443,67 @@ private struct ActionToken {
         return ancestor.lastToken == previous ? nil : ancestor.lastToken
       }
   }
+
+  static private func expectedResult(for token: TokenSyntax) -> ExpectedResult? {
+    guard token.isReference else { return nil }
+    guard let parent = token.parent else { return nil }
+    if let parent = parent.as(IdentifierExprSyntax.self), parent.identifier == token {
+      if let refArgs = parent.declNameArguments {
+        let name = SwiftName(base: token.text, labels: refArgs.arguments.map{ $0.name.text })
+        return ExpectedResult(name: name, kind: .reference)
+      }
+      if let (kind, callArgs) = getParentArgs(of: parent) {
+        let name = SwiftName(base: token.text, labels: callArgs)
+        return ExpectedResult(name: name, kind: kind)
+      }
+    }
+    if let parent = parent.as(MemberAccessExprSyntax.self), parent.name == token {
+      if let refArgs = parent.declNameArguments {
+        let name = SwiftName(base: token.text, labels: refArgs.arguments.map{ $0.name.text })
+        return ExpectedResult(name: name, kind: .reference)
+      }
+      if let (kind, callArgs) = getParentArgs(of: parent) {
+        let name = SwiftName(base: token.text, labels: callArgs)
+        return ExpectedResult(name: name, kind: kind)
+      }
+    }
+    let name = SwiftName(base: token.text, labels: [])
+    return ExpectedResult(name: name, kind: .reference)
+  }
+
+  static func getParentArgs<T:SyntaxProtocol>(of syntax: T) -> (kind: ExpectedResult.Kind, args: [String])? {
+    guard let parent = syntax.parent else { return nil }
+    if let call = parent.as(FunctionCallExprSyntax.self) {
+      // Check for: Bar.foo(instanceOfBar)(x:1)
+      if call.argumentList.count == 1 &&
```

Comment:
This is checking the innermost call around `foo` (`foo()`) which has no arguments rather than 1, so we won't match this condition and will use the this call's arg `()` rather than the outer one's `(_:_:)` to determine the symbol name, which is correct in this case.